### PR TITLE
[HWORKS-704][HWORKS-705][HWORKS-354]

### PR DESCRIPTION
**Pull request by SirOibaf**
…doesn't exists

### CODE REVIEW

File: templates/default/sql/dml/0.1.0.sql.erb [hunk 1]
```diff
@@ -53,11 +53,6 @@ REPLACE INTO `user_group` (`uid`, `gid`) VALUES (10001, 1007);
 REPLACE INTO variables(`id`, `value`) VALUES ("twofactor_auth", "<%= node['hopsworks']['twofactor_auth'] %>");
 REPLACE INTO variables(`id`, `value`) VALUES ("twofactor-excluded-groups", "<%= node['hopsworks']['twofactor_exclude_groups'] %>");
 
--- kmon
-
-INSERT INTO hosts(hostname, host_ip) VALUES <%= @hosts %> ON DUPLICATE KEY UPDATE cores=NULL;
```
Comment (kouzant) [inline templates/default/sql/dml/0.1.0.sql.erb:58]: We still need the hosts table, don't we?

**Review by kouzant — CHANGES_REQUESTED** [review]

File: templates/default/sql/dml/0.1.0.sql.erb (hunk 1 above)
Comment (SirOibaf) [reply templates/default/sql/dml/0.1.0.sql.erb:58]: I didn't remove the host table. I just removed the insert - I also changed Hopsworks to create the host if it doesn't exists: https://github.com/logicalclocks/hopsworks-ee/pull/1524

File: templates/default/sql/dml/0.1.0.sql.erb [hunk 2]
```diff
@@ -72,30 +67,8 @@ REPLACE INTO variables(`id`, `value`) VALUES
...
 
-REPLACE INTO variables(`id`, `value`) VALUES ("rm_ip", "<%= @rm_ip %>");
-
-REPLACE INTO variables(`id`, `value`) VALUES ("rm_port", "<%= @rm_port %>");
-
-REPLACE INTO variables(`id`, `value`) VALUES ("epipe_ip", "<%= @epipe_ip %>");
-
-REPLACE INTO variables(`id`, `value`) VALUES ("livy_ip", "<%= @livy_ip %>");
-
-REPLACE INTO variables(`id`, `value`) VALUES ("yarn_ui_ip", "<%= @yarn_ui_ip %>");
```
Comment (maismail) [inline templates/default/sql/dml/0.1.0.sql.erb:91]: not urgent, but we should also clean up this on in ec2-init as well 
https://github.com/logicalclocks/ec2-init/blob/master/ec2init/plans/preinit/pre_inits.py#L387

File: templates/default/sql/dml/0.1.0.sql.erb (hunk 1 above)
Comment (maismail) [reply templates/default/sql/dml/0.1.0.sql.erb:58]: that should be fine, we already delete the old hosts and re-register new head when initialising the cluster 
https://github.com/logicalclocks/ec2-init/blob/master/ec2init/plans/preinit/pre_inits.py#L592

**Review by maismail — APPROVED** [review]

**Pull request merged**